### PR TITLE
Add GoogleIamDAO.testIamPermission

### PR DESCRIPTION
**Pull request by eap**
This PR adds the testIamPermission method to GoogleIamDAO and child classes.

**PR checklist**
- [ ] For each library you've modified here, decide whether it requires a major, minor, or no version bump. (Click [here](/broadinstitute/workbench-libs/blob/develop/CONTRIBUTING.md) for guidance)
Determination: no version bump. Reason: addition of new public functions on classes.

In all cases:
- [x] Replace the appropriate version hashes in `README.md` and the `CHANGELOG.md` for any libs you changed with `TRAVIS-REPLACE-ME` to auto-update the version string
- [ ] Get two thumbsworth of PR review
- [ ] Verify all tests go green (CI _and_ coverage tests)
- [ ] Squash commits and **merge to develop**
- [ ] Delete branch after merge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Latest SBT dependency: `"org.broadinstitute.dsde.workbench" %% "workbench-metric
 
 Contains utility functions for talking to Google APIs and DAOs for Google PubSub, Google Directory, Google IAM, and Google BigQuery. 
 
-Latest SBT dependency: `"org.broadinstitute.dsde.workbench" %% "workbench-google" % "0.16-f339f30"`
+Latest SBT dependency: `"org.broadinstitute.dsde.workbench" %% "workbench-google" % "0.16-TRAVIS-REPLACE-ME"`
```
Comment (rtitle) [inline README.md:42]: Thanks for finding and doing this trick. :) I think there's a line below this (L46) that also needs to be TRAVIS-REPLACE-ME'd.

Comment (eap) [reply README.md:42]: Done

File: model/src/main/scala/org/broadinstitute/dsde/workbench/model/google/GoogleModel.scala [hunk 2]
```diff
@@ -22,6 +22,10 @@ case class ServiceAccountKeyId(value: String) extends ValueObject
 case class ServiceAccountPrivateKeyData(value: String) extends ValueObject with Base64Support
 case class ServiceAccountKey(id: ServiceAccountKeyId, privateKeyData: ServiceAccountPrivateKeyData, validAfter: Option[Instant], validBefore: Option[Instant])
 
+// Test IAM types.
+case class IamPermission(value: String) extends ValueObject
+case class IamTestResponse(permissions: Option[List[IamPermission]])
```
Comment (rtitle) [inline model/src/main/scala/org/broadinstitute/dsde/workbench/model/google/GoogleModel.scala]: Seems like `IamTestResponse` may not be used? If not then delete it.

**Comment by rtitle** [comment]
Looks good, a couple minor comments above. Will check Leo PR next.

File: model/src/main/scala/org/broadinstitute/dsde/workbench/model/google/GoogleModel.scala (hunk 2 above)
Comment (eap) [reply model/src/main/scala/org/broadinstitute/dsde/workbench/model/google/GoogleModel.scala]: Good catch, also, looks like I should have updated the changelog and version hash for workbench-model. That is also done now.

**Review by rtitle — APPROVED** [review]

**Pull request merged**